### ISSUE TITLE
Knowledge Graph output of model results

### ISSUE THREAD

**Issue by Dlux804** [Dlux804/McQuade-Chem-ML#16]
**Is your feature request related to a problem? Please describe.**
Current output of model results and parameters is complicated and does not fit into a spread sheet format.  

**Describe the solution you'd like**
Output of model results as a graph.  Model run is an entity that has subnodes of featurization method, data, learning algorithm.  Those would feed into results.  

**Describe alternatives you've considered**
I have stuck all the outputs, minus the PvA graph into a csv and it is really ugly.  It has a csv inside a csv...

**Additional context**
![Model_Results_Graph](https://user-images.githubusercontent.com/46574666/71194487-fec55380-2259-11ea-8a1a-f2e581872036.jpg)

**Comment by Dlux804** [comment]
@andreshyer This is what I want to talk to you about tomorrow.

**Comment by Dlux804** [comment]
Must adjust csv output so that the PvAM data is not a truncated dataframe printout.

**Comment by qle2** [comment]
@Dlux804 Please checkout model_kg.py located in branch quang_neo4j in folder neo4j. Instructions is in the code

**Comment by Dlux804** [comment]
@qle2 Unfortunately, I will do not have time right now to look into your code and spin it up.  Can you summarize what you have accomplished?  Pictures would be very helpful.

**Comment by qle2** [comment]
I have successfully created a knowledge graph using all of your machine learning results without running into the previous problems (missing nodes, no relationships, ...etc). It was all done in Python and it can definitely be standardized since all of our CSVs will look the same. 

Everything:
![image](https://user-images.githubusercontent.com/56049656/73504193-8c897800-439c-11ea-868e-6eb8e2977850.png)

Single graph:
![image](https://user-images.githubusercontent.com/56049656/73504221-aa56dd00-439c-11ea-80f9-ec83a61d6e06.png)

Problems that can be fixed quickly:
- If a list is stored as a value for a node (feature_list), the list will not show up as it usually does if you hover over the node. I think we need to find a better way to store feature_list. Example is provided below:

Feature list doesn't show up on the bottom of the GUI even though it is short.
![image](https://user-images.githubusercontent.com/56049656/73504680-31f11b80-439e-11ea-8fb8-6ea544d06a37.png)

But something as long as the regressor string does:
![image](https://user-images.githubusercontent.com/56049656/73504749-649b1400-439e-11ea-9647-018ed37f6c94.png)

- I only put in place holder phrases or words as the nodes' relationships. We can think of a standardize way to name the relationships.
- Right now, the some nodes have the values as their name instead of what they're actually called. For example: In the picture above, you can see that the node [4] is pointing to 0.84140. It should be feat_meth->[4]->feat_time->0.84140. The reason I don't think this can be done right now is because there is no column in the csv that has the intended names of those values to point to first. This might be a problem of aesthetics but since we're using a visualization tool, I think this should be addressed.

![image](https://user-images.githubusercontent.com/56049656/73504829-ab890980-439e-11ea-9768-4396325cd972.png)


Potential problem:
- You need to open up Neo4j Desktop and open a graph locally for the code to work since py2neo needs to be pointing to a localhost in order to upload its nodes and relationships. This means that we need to have Neo4j Desktop running constantly if you want to make everything from running ML to visualizing the graph to be high-throughput.
